package.json

```diff
@@ -37,6 +38,7 @@
   "license": "UNLICENSED",
   "private": true,
   "dependencies": {
+    "@streamr/streamr-layout": "^1.1.0",
```

Comment:
`https://github.com/streamr-dev/streamr-layout/releases` should we be using the latest version instead?